db_stress_tool/db_stress_common.h

```diff
@@ -161,6 +161,7 @@ DECLARE_int32(range_deletion_width);
 DECLARE_uint64(rate_limiter_bytes_per_sec);
 DECLARE_bool(rate_limit_bg_reads);
 DECLARE_bool(use_txn);
+DECLARE_uint64(txn_write_policy);
```

Comment:
Is it easier to understand if we use string here, and use a mapping from string to policy?

Reply:
Makes sense. Let me leave it as a TODO.